schema/graph.json

```diff
@@ -13,7 +13,35 @@
       "additionalProperties": true,
       "properties": {
         "name": {
-          "type": "string"
+          "type": "string",
+          "description": "Name of the graph"
+        },
+        "environment": {
+          "type": "object",
+          "description": "Information about the execution environment for the graph",
+          "additionalProperties": true,
+          "required": [
+            "type"
```

Comment:
`type` cannot be mandatory? Many existing graphs don't have it, and for instance `fbp` does not provide it either unless it is in the .fbp input file

Reply:
@jonnor I set it as required _if_ you have `environment` object (which itself is not required)

Reply:
OK